go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/coreos/go-oidc
```

Comment:
Last time we did this we broke imports of this branch.

https://github.com/coreos/go-oidc/issues/230
https://github.com/coreos/go-oidc/pull/231

I don't know if that's changed in the upstream Go tooling since then.

Reply:
all the go.mod and vendor bits are in TMP commits I would drop before we merge this, I just included those so we could actually see the effective change in the dependency code the square → go-jose change would make

if you're done looking at the vendor change, I'll drop those commits out completely

Reply:
I agree we don't want to add go.mod to the v2 branch

Reply:
done, dropping TMP commmits